src/browser/web-viewer.js

```diff
@@ -60,6 +69,50 @@ define((require, exports, module) => {
     })
   });
 
+
+  const fetchScreenshot = iframe =>
+    fromDOMRequest(iframe.getScreenshot(100 * devicePixelRatio,
+                                        62.5 * devicePixelRatio,
+                                        'image/png'));
+
+  const fetchThumbnail = uri => new Promise((resolve, reject) => {
+    const request = new XMLHttpRequest();
```

Comment:
Just add a comment here to say that this bad, we know it's bad, and we'll fix that later.